### PR TITLE
Display the user's avatar when they shared their location

### PR DESCRIPTION
**Pull request by andybalaam**
Builds on https://github.com/matrix-org/matrix-react-sdk/pull/7423 .  This just adds one commit: [`59e6671`](https://github.com/matrix-org/matrix-react-sdk/pull/7424/commits/59e667129901fe0702582a3befdfe55972dea03a)

image

---
Here's what your changelog entry will look like:

## ✨ Features
 * Display the user's avatar when they shared their location ([\#7424](https://github.com/matrix-org/matrix-react-sdk/pull/7424)). Contributed by @andybalaam.

Preview: https://61c1eb8ba7ed89102699bfff--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

**Review by gaelledel — APPROVED** [review]
Can we also display the user avatar before they share their location
Like that?
Screenshot 2021-12-21 at 12 32 05
?

**Comment by andybalaam** [comment]
> Can we also display the user avatar before they share their location

Yes - I'll add it to the story about the before-sharing dialog.

File: src/components/views/context_menus/MessageContextMenu.tsx [hunk 1]
```diff
@@ -122,7 +126,11 @@ export default class MessageContextMenu extends React.Component<IProps, IState>
         // HACK: Intentionally say we can't pin if the user doesn't want to use the functionality
         if (!SettingsStore.getValue("feature_pinning")) canPin = false;
 
-        this.setState({ canRedact, canPin });
+        const isLoc = isLocationEvent(this.props.mxEvent);
+        const canForward = !isLoc;
+        const canShare = !isLoc;
+
+        this.setState({ canRedact, canPin, canForward, canShare });
```
Comment (t3chguy) [inline src/components/views/context_menus/MessageContextMenu.tsx:133]: merge went wrong, undid the review changes

**Review by t3chguy — CHANGES_REQUESTED** [review]

File: src/components/views/context_menus/MessageContextMenu.tsx (hunk 1 above)
Comment (andybalaam) [reply src/components/views/context_menus/MessageContextMenu.tsx:133]: Ah, I'm really sorry for letting that slip through.

Comment (andybalaam) [reply src/components/views/context_menus/MessageContextMenu.tsx]: Fixed in d17ed30aa1

**Review by t3chguy — APPROVED** [review]

**Pull request merged**